### ISSUE TITLE
The Mod is (Not Responding)

### ISSUE THREAD

**Issue by TGS963** [Backdash/MonikaModDev#7970]
I had MAS open and it went into not responding for about an hour, after which I decided to reboot and relaunch. Turns out its still like that. I tried playing other mods and they work fine. I tried to reinstall MAS with a fresh copy of DDLC files but the problem persists. I have also tried switching to an older persistent file but nothing works... plz help

**Comment by ThePotatoGuy** [comment]
Couple of things:
* system specs?
* are you out of disk space? 
* were you out of RAM?

**Comment by TGS963** [comment]
> Couple of things:
> 
> * system specs?
> * are you out of disk space?
> * were you out of RAM?

My system specs are:
CPU: AMD Ryzen 5 2600x 6 core
GPU: MSI Radeon RX 5600xt 6GB
RAM 16GB
 
I still have 40 GB left on my main SSD so I don't yhink I'm running out.

No I don't think I was out of RAM, I don't game or anything while MAS is open. I mostly watch videos or study.

P.S. Something I found today was that MAS automatically goes not responding and crashes and there is a 0kb persistent.tmp in the MAS appdata folder. I can't delete that either. If I boot into another OS and delete that file, MAS runs on the next reboot. But crashes randomly in a similar way

**Comment by ThePotatoGuy** [comment]
can you upload your persistent files here?

**Comment by TGS963** [comment]
Yes, I do not have the persistent.tmp right now because I deleted it from a linux installation to make MAS work again (I don't think windows would even allow me to upload that considering it did not let me even copy that file). I have the tmp file from a pervious backup I took when MAS would launch as Not Responding. I will upload that and the persistent files from now.

[Monika After Story(Current).zip](https://github.com/Monika-After-Story/MonikaModDev/files/7272729/Monika.After.Story.Current.zip)
[Monika After Story(Not Responding).zip](https://github.com/Monika-After-Story/MonikaModDev/files/7272730/Monika.After.Story.Not.Responding.zip)
